site/src/lib/api/model/user.model.ts

```diff
@@ -234,6 +234,7 @@ export class User {
     updatedProperties: Partial<UserProperties>,
   ): Promise<User> {
     if (
+      this.shortname &&
       updatedProperties.shortname &&
       updatedProperties.shortname !== this.shortname
     ) {
```

Comment:
This check should only be performed when the user already has a `shortname`. Otherwise it prevents new users from setting their `shortname` for the first time as part of the `completeSignup` Next API endpoint.